CHANGELOG.md

```diff
@@ -165,6 +165,9 @@ Note: We're not following semantic versioning yet, we are going to talk about th
   and footer (the OGL logo) by marking them as non-focusable elements
   ([PR #774](https://github.com/alphagov/govuk-frontend/pull/774))
 
+- Update CSS modifier class for a bold label
```

Comment:
I don't think I'd know what this meant if I read this without looking at the PR.

What about something like "Use the correct class name in bold label example (`govuk-label--s` rather than `govuk-label--bold`)"

Reply:
yeah, was a bit optimistic with the title. updated